sdk/synapse/Azure.Analytics.Synapse.Artifacts/tests/DatasetClientLiveTests.cs

```diff
@@ -39,7 +40,8 @@ await foreach (var expectedDataset in DatasetClient.GetDatasetsByWorkspaceAsync(
         [Test]
         public async Task TestCreateDataset()
         {
-            var operation = await DatasetClient.StartCreateOrUpdateDatasetAsync("MyDataset", new DatasetResource(new Dataset(new LinkedServiceReference(LinkedServiceReferenceType.LinkedServiceReference, "testsynapseworkspace-WorkspaceDefaultStorage"))));
+            string workspaceName = TestEnvironment.WorkspaceUrl.Split('.')[0].Split('/')[2];
+            var operation = await DatasetClient.StartCreateOrUpdateDatasetAsync("MyDataset", new DatasetResource(new Dataset(new LinkedServiceReference(LinkedServiceReferenceType.LinkedServiceReference, workspaceName + "-WorkspaceDefaultStorage"))));
```

Comment:
I recommend using your `Recording` object to get a random name. The seed is stored, so it will be consistent across runs, but also helps avoid conflicts should the test fail and the resource not get cleaned up, or if there are any similar names in concurrently-ran tests.

Reply:
Also, we generally try to stick with explicit variable types, especially in samples (I see that below as well) instead of `var`.